README.md

```diff
@@ -3,3 +3,10 @@
 ## Scripts for formatting and uploading Food Distribution Organization and Food-handout Site data dumps (.CSV, .xlsx, etc.) into the [National Free Food Data Collection Airtable](https://airtable.com/invite/l?inviteId=invgFbPkoS2sXfYoi&inviteToken=edaa3e00328ce2a8c0bf160c51210a071c06da6edb6b79e06773b6c1063bd325)
 
 Join the conversation in our [Slack workspace](https://join.slack.com/t/covid-basic-needs), To-do's are in the [#upload-convert-to-airtable channel](https://covid-basic-needs.slack.com/archives/C0120CGRABV) (To-do's are tracked via [Workstreams](workstreams.ai) - an integrated Slack App)
+
+
+## Steps to get started:
+1. Clone the repo.
+2. Run `npm install`.
```

Comment:
I was using Yarn (has a few caching benefits) as opposed to NPM. If you don't have it we can stick with NPM, if you do I think we'd prefer Yarn. _(if NPM, where's the package-lock.json?)_